ngraph_bridge/ngraph_utils.cc

```diff
@@ -463,6 +475,198 @@ bool IsProcessedByNgraphPass(Graph* g) {
   return false;
 }
 
+Status DumpNGraph(int file_idx, tensorflow::GraphDef* graph_def,
```

Comment:
So this dumps an event right?
Lets rename this function to reflect that.

We already have an existing function called `DumpGraphs`, so we do not want to confuse with that